CONTRIBUTING.md

```diff
@@ -324,13 +335,12 @@ Good docs make developers happy, and we love happy developers! We've got a few d
 
 Bonus: **Know when to break the rules**. Documentation writing is as much art as it is science. Sometimes you will have to deviate from these rules in order to write good documentation.
 
-[MyST](https://myst-parser.readthedocs.io/en/latest/) Elements Usage
+#### [MyST](https://myst-parser.readthedocs.io/en/latest/) Elements Usage
 
 1. Use the `{tab}` element to show multiple ways of doing one thing. [Example](https://docarray.jina.ai/fundamentals/document/#document) 
 2. Use the `{admonition}` boxes with care.
 3. Use `{dropdown}` to hide optional content, such as long code snippets or console output.
 
-
 Note: Example code in the documentation should also follow our [code style conventions]().
```

Comment:
you u remove this section. it is not relevant anymore as we don't use myst

Reply:
Roger that. What about the `[code style conventions]()`? There's no URL

Reply:
you can remove the link, lets just say follow our code style